docs/sanic/exceptions.md

```diff
@@ -43,3 +43,11 @@ Some of the most useful exceptions are presented below:
   usually occurs if there is an exception raised in user code.
 
 See the `sanic.exceptions` module for the full list of exceptions to throw.
+
+You can also use `exceptions_base`, it can catch the exception just like
+normal catching. It can be used for catching all unexpected error and eliminate
+the risk that server information being stolen by attackers/
```

Comment:
@asdfghjBennyHuang 
Do you put `debug=True` in `app.run()`?
The following code like your test code does not display tracing.


```
from sanic import Sanic
from sanic.response import text

app = Sanic(__name__)


@app.route('/')
async def test(request):
    a = [1, 2, 3]
    return text(a[5])


if __name__ == '__main__':
    app.run(host='127.0.0.1', port=8000)
```

Would you post the code which display tracing?